src/SwedbankPay/Core/Library/Methods/Checkout.php

```diff
@@ -85,6 +86,55 @@ public function initiatePaymentOrderPurchase($orderId, $consumerProfileRef = nul
         return $result;
     }
 
+    /**
+     * Initiate Payment Order Recurrent Payment
+     *
+     * @param mixed $orderId
+     * @param string $recurrenceToken
+     *
+     * @return Response
+     * @throws \Exception
+     */
+    public function initiatePaymentOrderRecur($orderId, $recurrenceToken)
```

Comment:
`initiatePaymentOrderRecur` feels like something that should be inside the SDK, not in Core.

Reply:
We wrap some helpful method and proxy it to the SDK. Don' t worry, we will do it in the SDK also